docs/meetings/2022/TechArea20220823.md

```diff
@@ -45,19 +41,19 @@
 
 ### Support Update
 
--   FIU (Mat) working with FIU to resolve connection issues while attempting to mirror OSG Software repos.
-
--   FNAL (Carl) worked with Yuijun and Kevin Retzke on Gratia probe changes. 
-    Software team work is done for now;
-    at this point Yuijun and Kevin should discuss their requirements.
+...
 
 ## OSG Release Team
 
+-   TimT: new condor releases (9.11.1)
+
 -   Ready for Testing
-    -   xrootd-monitoring-shoveler 1.1.2
-    -   HTCondor 9.0.16 (important bug fixes)
-    -   Upcoming: HTCondor 9.11.0 RC2
+    -   gratia-probe
+    -   stashcp
 
 ### Discussion
 
-    - Release xrootd-monitoring-shoveler and HTCondor 9.0.16 this week.
+-   TimT: note prce2 more strict about character classes in regex (hyphen must go at the end `[abc-]`)
+-   BrianL: need to find victims to update from stable series to 9.12
+-   AI (TimT): reach out to John Thiltges about feasibility of updating to 10.0?
+    (TimT: in a couple weeks)
```

Comment:
```suggestion
    (TimT: 9.12.0 in a couple weeks)
```

Reply:
Ah i guess i misunderstood and thought he was referring to 10.0.